src/Microsoft.Framework.DependencyInjection.Abstractions/project.json

```diff
@@ -8,7 +8,7 @@
         "net45": {
         },
         "dnx451": { },
```

Comment:
Can `dnx451`, `net45` also be removed if we have `dotnet` listed (and we aren't using them to `#ifdef` in the code?)